src/Illuminate/Auth/AuthServiceProvider.php

```diff
@@ -48,7 +49,7 @@ protected function registerAuthenticator()
 	 */
 	protected function registerUserResolver()
 	{
-		$this->app->bind('Illuminate\Contracts\Auth\Authenticatable', function($app)
+		$this->app->bind(Authenticatable::class, function($app)
```

Comment:
It should do. `'\Illuminate\Contracts\Auth\Authenticatable'` technically isn't the name of that class though. The fully qualified name is without the slash.